presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -334,7 +335,11 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
                 featuresConfig,
                 typeRegistry,
                 blockEncodingManager,
-                new SessionPropertyManager(new SystemSessionProperties(new QueryManagerConfig(), new TaskManagerConfig(), new MemoryManagerConfig(), featuresConfig)),
+                new SessionPropertyManager(new SystemSessionProperties(new QueryManagerConfig(),
```

Comment:
wrap `new QueryManagerConfig()`.

We prefer either one of the style

```
doSomethign(parameter1, parameter2)
```

or 

```
doSomethign(
  parameter1, 
  parameter2)
```

We are trying to avoid the

```
doSomethign(parameter1, 
  parameter2)
```
style across the codebase